### PR TITLE
Remove ternary condition due terraform limitations (CORE-421)

### PR DESCRIPTION
**Pull request by kobrikx**
#### What's new:

 - Removed one condition for https due ternary terraform limitations

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -690,3 +690,8 @@ variable "cpu_architecture" {
   description = "When you register a task definition, you specify the CPU architecture. The valid values are X86_64 and ARM64"
   default     = "X86_64"
 }
+
+variable "force_delete" {
```
Comment (igorkotof) [inline variables.tf]: @kobrikx can we rename to `ecr_force_delete`?

**Review by igorkotof — CHANGES_REQUESTED** [review]
Have you tested it? there is opposite PR from @AutomationD (https://github.com/hazelops/terraform-aws-ecs-app/pull/28/files) ?

**Comment by DimmKirr** [comment]
It removes my PR https://github.com/hazelops/terraform-aws-ecs-app/pull/28/files#diff-dc46acf24afd63ef8c556b77c126ccc6e578bc87e3aa09a931f33d9bf2532fbbR23 which fixes the issue in question.

**Comment by kobrikx** [comment]
> It removes my PR https://github.com/hazelops/terraform-aws-ecs-app/pull/28/files#diff-dc46acf24afd63ef8c556b77c126ccc6e578bc87e3aa09a931f33d9bf2532fbbR23 which fixes the issue in question.

@AutomationD 
If you don't want to remove it - we will need to somehow screen(shield) or remake it. When I remove tls - there are no errors. So removing tls - resolve my error :) 
Any ideas?

**Review by igorkotof — APPROVED** [review]
LGTM for now, let's monitor

**Pull request merged**